pipeline_dp/budget_accounting.py

```diff
@@ -172,19 +174,21 @@ def compute_budgets(self):
 
         total_weight_eps = total_weight_delta = 0
         for mechanism in self._mechanisms:
-            total_weight_eps += mechanism.weight
-            if mechanism.mechanism_spec.use_delta():
-                total_weight_delta += mechanism.weight
+            for i in range(mechanism.mechanism_spec.count):
+                total_weight_eps += mechanism.weight
+                if mechanism.mechanism_spec.use_delta():
+                    total_weight_delta += mechanism.weight
 
         for mechanism in self._mechanisms:
             eps = delta = 0
-            if total_weight_eps:
-                numerator = self._total_epsilon * mechanism.weight
-                eps = numerator / total_weight_eps
-            if mechanism.mechanism_spec.use_delta():
-                if total_weight_delta:
-                    numerator = self._total_delta * mechanism.weight
-                    delta = numerator / total_weight_delta
+            for i in range(mechanism.mechanism_spec.count):
```

Comment:
No need in this loop. 

**Explanation:** `eps/delta` in `MechanismSpec` is per one application of the mechanism, i.e. no need in count